### ISSUE TITLE
test failure from Getopt::Long

### ISSUE THREAD

**Issue by karenetheridge** [petdance/ack2#564]
The dependency on Getopt::Long needs to be increased?  GetOptionsFromString is not present in version 2.35.

(This failure is also being submitted via cpantesters.)

```
cpanm (App::cpanminus) 1.7038 on perl 5.008008 built for darwin-2level
...
Checking dependencies from MYMETA.json ...
Checking if you have Getopt::Long 2.35 ... Yes (2.35)
...
t/noenv......................
#   Failed test at t/noenv.t line 97.
#                   'Undefined subroutine &Getopt::Long::GetOptionsFromString called at /Volumes/amaretto/Users/ether/.cpanm/work/1437060628.20948/ack-2.14/blib/lib/App/Ack/ConfigLoader.pm line 593.'
#     doesn't match '(?-xism:--noenv found in (?:.*)[.]ackrc)'
# [
#   'Undefined subroutine &Getopt::Long::GetOptionsFromString called at /Volumes/amaretto/Users/ether/.cpanm/work/1437060628.20948/ack-2.14/blib/lib/App/Ack/ConfigLoader.pm line 593.'
# ]
# Looks like you failed 1 test of 5.
dubious
        Test returned status 1 (wstat 256, 0x100)
DIED. FAILED test 5
        Failed 1/5 tests, 80.00% okay
```

**Comment by petdance** [comment]
This is already noted in the issues database.  #281

**Comment by dolmen** [comment]
Fixed by commit 9f6c3882c2088d261f00a7789134868c0b0e767f: Getopt::Long dependency upgraded to 2.38.

**Comment by petdance** [comment]
This went out in 2.16.
